framework/wazuh/common.py

```diff
@@ -75,7 +75,7 @@ def set_paths_based_on_ossec(o_path='/var/ossec'):
     AUTHD_SOCKET = "{0}/queue/ossec/auth".format(ossec_path)
 
 # Agent upgrading variables
-wpk_repo_url = "packages.wazuh.com/wpk/"
+wpk_repo_url = "https://packages.wazuh.com/wpk/"
```

Comment:
The protocol was removed when the `--http` parameter was added (https://github.com/wazuh/wazuh/pull/798). It is later added in this part of the code:
https://github.com/wazuh/wazuh/blob/458d330c3e2dfb186f85754292f6a5a7ac7ddbd2/framework/wazuh/agent.py#L1922-L1923